libsepol/patches/dont-build-utils.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/Makefile b/Makefile
+index 5b5d03e2..5744d7a4 100644
+--- a/Makefile
++++ b/Makefile
+@@ -4,26 +4,21 @@ export DISABLE_CIL
+ 
```

Comment:
if we don't want to maintain patches, we can also let it build, but then do `rm -rf /rootfs/usr/....`

Reply:
They aren't likely to change that often. Maybe I should send a patch upstream so there's an option like SUBDIRS in libselinux

Reply:
Upstreamed a couple of change for SUBDIRS in every lib.